common/src/TrenchBroomApp.cpp

```diff
@@ -112,6 +112,37 @@ namespace TrenchBroom {
                 return;
             }
 
+            // initialize stylesheets
+            QString styleSheetString;
+
+            // load base stylesheet
+            QFile baseStyle = QFile(IO::pathAsQString(IO::SystemPaths::appDirectory() + IO::Path("styles/base.qss")));
+            assert(baseStyle.exists());
+            baseStyle.open(QFile::ReadOnly | QFile::Text);
+            QTextStream baseStyleStream(&baseStyle);
+            styleSheetString += baseStyleStream.readAll() + "\n";
+            baseStyle.close();
+
+            // load override stylesheets
+            QDir styleDir(IO::pathAsQString(IO::SystemPaths::appDirectory() + IO::Path("styles")), "*.qss", QFlags<QDir::SortFlag>(QDir::Name | QDir::IgnoreCase), QDir::Files);
+            foreach(QString str, styleDir.entryList()) {
```

Comment:
What is `foreach`? Why not a range-based for loop as usual?

Reply:
`foreach` is a Qt macro that roughly corresponds to `for (QStringList::const_iterator it = container.begin(), end = container.end(); it != end; ++it)`

Though looking into it further it appears to be discouraged post-Qt 5.7 in favor of C++98/C++11 range-for, so this is a case of my practices needing an update. Will switch it over to something more standard.

Reply:
Yes, please use range-based for, since that's what we commonly use in TB. Thanks!